source/_integrations/matter.markdown

```diff
@@ -23,182 +23,241 @@ ha_platforms:
+[Eve Motion on Amazon](https://amzn.to/3jDujiP)
 
-- [Eve Energy on Amazon](https://amzn.to/3YuO62P)
-- [Eve Door & Window on Amazon](https://amzn.to/3RIU6ml)
-- [Eve Motion on Amazon](https://amzn.to/3jDujiP)
+### Nanoleaf Matter bulbs and Lightstrips
+
+- Although the products work great once commissioned, multiple users have reported that commissioning them can be a bit difficult and requires some patience and multiple resets or optimizations to your home network.
+- Check the [Nanoleaf Matter infopage](https://nanoleaf.me/en-EU/integration/matter/) for all supported products and instructions.
+
+### Philips Hue (V2) Bridge
+
+The Philips Hue V2 bridge supports Matter since a recent update (the beta program closed, it is now officially available). You can enable Matter support from the Hue app after which you can commission it to Home Assistant and other fabrics.
+
+- It does not make a lot of sense to bind the Hue bridge to Home Assistant because you will loose functionality over the default Hue integration in Home Assistant, such as button press events and (dynamic) scenes.
+- You will need a Hue/Signify (cloud) account and the app before you can use Matter.
+- Device events for example for dimmer remotes are not supported.
+- Only basic control of lights is supported, no scenes, events, effects etc.
+
+### TP-Link Tapo P125M (power plug)
+
+- Look for the M addition in the model name, a device without the M (regular P125) is not Matter compliant.
+- This device is available in the US only.
+
+[TP-Link Tapo P125M on Amazon](https://amzn.to/3RILJah)
 
 ## Troubleshooting
 
-### I do not see the option to add a Matter device in the settings
-We've added the option to **Add a Matter device** from the **Settings**>**Devices** in a recent version of the Home Assistant frontend, available from version 2023.3 and upwards or if you're running the Home Assistant nightly channel. If you are on a previous version of Home Assistant, you should see a button **Configure** on the Matter integration card within **Settings**>**Devices & Services**>**Integrations**. Click that **Configure** button and you should be able to see the **Commission** button on the Companion app.
+### General recommendations
 
-### I do not see the button "Commission using the Companion app"
-This button will only be visible within the Home Assistant Companion App (so not in the browser) and your device meets all requirements for Matter support.
-- For iOS, minimum version is iOS 16 (minimal 16.3 is preferred) and the most recent version of the HA companion app.
-- For Android, minimum version is 8.1 and the most recent version of both the Google Home app and the (full) HA Companion app, downloaded from the app store.
+- Using Thread based Matter devices in Home Assistant requires Home Assistant OS (version 10 and above) because of kernel patches to solve routing issues. Not using HAOS (and thus the official Matter add-on) is at your own risk.
```

Comment:
```suggestion
- Using Thread-based Matter devices in Home Assistant requires Home Assistant OS (version 10 and above) because of kernel patches to solve routing issues. Not using HAOS (and thus the official Matter add-on) is at your own risk.
```